stock_lot_condition/demo/stock_lot_condition_demo.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<odoo noupdate="1">
+    <record id="condition_good" model="stock.lot.condition">
```

Comment:
Please check if there's any standard classification for this, and at least I'm thinking this should have 4 levels.

Reply:
Yes, I mean four records. And I would like to have them as data if we start from a more or less standarized classification of the condition.

Reply:
Check https://share.google/aimode/y4CCbOYSGjgC4iRAV for possible classifications.